server/lib/betterThanFT.js

```diff
@@ -0,0 +1,27 @@
+const query = require('./database').query;
+const escape = require('mysql').escape;
+const debug = require('debug')('perf-widget:lib:betterThanFT');
+
+module.exports = function betterThanFT(name, value) {
+	const count = `SELECT COUNT(current_values.value) as amount FROM current_values JOIN page ON page.id = current_values.page_id JOIN properties ON properties.id = current_values.property_id WHERE current_values.page_id NOT IN (SELECT id FROM page WHERE domain='www.theguardian.com' OR domain='international.nytimes.com' OR domain='www.nytimes.com' OR domain='www.wsj.com') AND properties.name = ${escape(name)};`;
+	const average = `SELECT better, a.average FROM properties JOIN (SELECT AVG(current_values.value) as average FROM current_values JOIN page ON page.id = current_values.page_id JOIN properties ON properties.id = current_values.property_id WHERE current_values.page_id NOT IN (SELECT id FROM page WHERE domain='www.theguardian.com' OR domain='international.nytimes.com' OR domain='www.nytimes.com' OR domain='www.wsj.com') AND properties.name = ${escape(name)}) as a WHERE name = ${escape(name)};`;
```

Comment:
Maybe I should rename from domain to host, as host is what I am using and wanting